{{cookiecutter.pipeline_slug}}/environment.yml

```diff
@@ -1,9 +1,9 @@
-name: nfcore-{{ cookiecutter.pipeline_slug }}
+name: nfcore-{{ cookiecutter.pipeline_slug.replace('/', '-') }}
```

Comment:
Can simplify to `{{ cookiecutter.pipeline_slug }}` if implementing the above suggestion in `cookiecutter.json`

NB: Drop the `nfcore-` prefix.